common4j/src/main/com/microsoft/identity/common/java/providers/keys/ClientCertificateMetadata.java

```diff
@@ -55,7 +58,7 @@ public String getAlias() {
      * @return char[]
      */
     public char[] getPassword() {
-        return mPassword;
+        return mPassword == null ? null : mPassword.clone();
```

Comment:
Do we want to use clone here, or Arrays.copyOf() ?  I'd default to the Arrays.copyOf method.